raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -381,15 +343,11 @@ library NettingChannelLibrary {
         closer_amount = total_deposit - counter_amount;
 
         if (counter_amount > 0) {
-            if (!self.token.transfer(counter_party.node_address, counter_amount)) {
-                throw;
-            }
+            require(self.token.transfer(counter_party.node_address, counter_amount));
         }
 
         if (closer_amount > 0) {
-            if (!self.token.transfer(closing_party.node_address, closer_amount)) {
-                throw;
-            }
+            require(self.token.transfer(closing_party.node_address, closer_amount));
```

Comment:
Don't wrap those in `require`. They are external calls and as such the `if` check is fine.